tests/helpers/basic_models.py

```diff
@@ -349,6 +349,19 @@ def forward(self, x1: Tensor, x2: Tensor, x3: Tensor, scale: int):
         return self.model(scale * (x1 + x2 + x3))
 
 
+class BasicModel_ConvNet_Optim(nn.Module):
+    def __init__(self):
```

Comment:
I added the `-> None` type hint to all of the basic models a while back in the master branch, so you should add it here for when we do merge with the master branch.